features/wallet/impl/src/main/java/com/tangem/feature/wallet/presentation/wallet/viewmodels/intents/WalletCardClickIntents.kt

```diff
@@ -15,17 +24,48 @@ internal interface WalletCardClickIntents {
     fun onDeleteAfterConfirmationClick(userWalletId: UserWalletId)
 }
 
-internal class WalletCardClickIntentsImplementor @Inject constructor() : WalletCardClickIntents {
+internal class WalletCardClickIntentsImplementor @Inject constructor(
```

Comment:
Следуя договоренности про имплементации лучше сделать так:
```suggestion
internal class DefaultWalletCardClickIntents @Inject constructor(
```

Ещё предлагаю избавиться от слова `Click` в названиях интерфейсов/имплементаций, так как они не только намерения кликов содержат.